src/Elastic.Markdown/Helpers/DocumentationObjectPoolProvider.cs

```diff
@@ -17,7 +19,22 @@ internal static class DocumentationObjectPoolProvider
 	public static readonly ObjectPool<StringBuilder> StringBuilderPool = PoolProvider.CreateStringBuilderPool(256, 4 * 1024);
 	public static readonly ObjectPool<ReusableStringWriter> StringWriterPool = PoolProvider.Create(new ReusableStringWriterPooledObjectPolicy());
 	public static readonly ObjectPool<HtmlRenderSubscription> HtmlRendererPool = PoolProvider.Create(new HtmlRendererPooledObjectPolicy());
+	private static readonly ObjectPool<LlmMarkdownRenderSubscription> LlmMarkdownRendererPool = PoolProvider.Create(new LlmMarkdownRendererPooledObjectPolicy());
 
+	public static string UseLlmMarkdownRenderer(BuildContext buildContext, Action<LlmMarkdownRenderer> action)
```

Comment:
```suggestion
	public static string UseLlmMarkdownRenderer<TContext>(BuildContext buildContext, TContext context, Action<TContext, LlmMarkdownRenderer> action)
```

That way the invocations can be static e.g:

```csharp

DocumentationObjectPoolProvider.UseLlmMarkdownRenderer(context, document, static (d, r) =>
		{
			_ = r.Render(d);
		});
```

Ultra nitpicky not a blocker :)